stablehlo/reference/Ops.cpp

```diff
@@ -869,50 +893,39 @@ Tensor evalGatherOp(const Tensor &operand, const Tensor &startIndices,
                     ShapedType resultType) {
   Tensor result(resultType);
   Axes batchDims;
-  for (auto i = 0; i < result.getRank(); ++i)
-    if (!llvm::is_contained(offsetDims, i)) batchDims.push_back(i);
+  for (auto d : result.getAxes())
```

Comment:
Introduced getAxes to be used here and in evalReduceWindowOp. In #1401, we can explore having this as a free function `axes(...)`, similarly to other metafunctions.